### PR TITLE
adding starpolymers - first time contribution

### PR DESCRIPTION
**Pull request by debeshmandal**
Hi @conda-forge/help-python - This is my first contribution of a package to conda-forge so apologies if I have missed anything, I have tried to ensure that all the checkpoints are completed and that the recipe is valid. Thanks for your help

## Checklist:

- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by debeshmandal** [comment]
@conda-forge/core

After examining the azure pipelines, I've seen that there are conflicts with setting up the environment which I do not know how to fix - please can you advise on what I should do to fix this problem?

File: recipes/starpolymers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - pandas
+    - argparse
+    - matplotlib
```
Comment (scopatz) [inline recipes/starpolymers/meta.yaml:28]: This should be matplotlib-base

File: recipes/starpolymers/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - pytest
```
Comment (scopatz) [inline recipes/starpolymers/meta.yaml:29]: Is pytest a runtime dep?

File: recipes/starpolymers/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (scopatz) [inline recipes/starpolymers/meta.yaml:23]: Is this a python 3 lib?

File: recipes/starpolymers/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+    - numpy
+    - scipy
+    - pandas
+    - argparse
```
Comment (scopatz) [inline recipes/starpolymers/meta.yaml:27]: argparse is part of Python now :wink:

File: recipes/starpolymers/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    - matplotlib-base
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
```
Comment (scopatz) [inline recipes/starpolymers/meta.yaml:32]: please remove comments

**Pull request closed without merging**